Assets/GothicVR/Scripts/Creator/VobCreator.cs

```diff
@@ -350,9 +356,24 @@ public static void CreateItem(int itemId, GameObject go)
 
             CreateItemMesh(item, go);
         }
-        
+
+        /// <summary>
+        /// Render item inside GameObject
```

Comment:
Hm. I don't know if I get it. There are items which we spawn inside GOs? Can you name an example as comment, please.